extension/src/experiments/model/index.ts

```diff
@@ -128,7 +128,7 @@ export class ExperimentsModel extends ModelWithPersistence {
 
   public toggleStatus(id: string) {
     if (
-      this.flattenExperiments().find(({ id: queuedId }) => queuedId === id)
+      this.getFlattenedExperiments().find(({ id: queuedId }) => queuedId === id)
```

Comment:
[F] The main change in this file was to encapsulate access to experiments and checkpoints so that details (selected and starred) are always added.